### PR TITLE
docs: add browser polyfill for IE11 with hash-based routing

### LINKED ISSUE #26511: Angular docs: Suggest es7/array polyfill for hash-based routing
## I'm submitting a...

<pre><code>
[x] Documentation issue or request
</code></pre>

## Current behavior

Using hash-based routing strategy in IE11, angular throws an error:
<code>IE11 - Object doesn't support property or method 'includes' - javascript window.location.hash</code>

The solution is to add the <code>es7/array</code> polyfill to <code>polyfills.ts</code>:
https://stackoverflow.com/a/52630710/166850

## Expected behavior

Please add this polyfill to the table of polyfills in the Angular documentation.

## Environment

<pre><code>
Angular version: 6.0.0

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [x] IE version 11
- [ ] Edge version XX
 
Others:

</code></pre>

### PR DESCRIPTION
**Pull request by brandonroberts**
Closes #26511

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/browser-support.md [hunk 1]
```diff
@@ -372,6 +372,27 @@ Here are the features which may require additional polyfills:
...
+    <td>
+
+      [Router](guide/router) 
+      
+      when using hash-based routing
+    </td>
+
+    <td>
+
+      [ES7/array](guide/browser-support#core-es7-reflect)
```
Comment (trotyl) [inline aio/content/guide/browser-support.md]: FYI, the entire `esN` entry in core-js is in wrong version number and exists just for compatibility reason, it's not recommended to use and has been removed in v3.

Comment (trotyl) [reply aio/content/guide/browser-support.md]: Yep, I agree it's not caused by this PR, but those entries are also not recommended in 2.x version. Should either be `core-js/fn/array` for all Array methods or `core-js/fn/array/xxx` for specific one.

Comment (trotyl) [reply aio/content/guide/browser-support.md]: Also, why it's the link to be `core-es7-reflect` while the feature is about Array?

File: aio/content/guide/browser-support.md [hunk 2]
```diff
@@ -372,6 +372,27 @@ Here are the features which may require additional polyfills:
 
   </tr>
 
+  <tr style="vertical-align: top">
+
+    <td>
+
+      [Router](guide/router) 
+      
+      when using hash-based routing
```
Comment (IgorMinar) [inline aio/content/guide/browser-support.md]: Can you add a link to api doc or a guide that clarifies what this means. We want people to add this only if they absolutely need it.

**Review by IgorMinar — APPROVED** [review]
Please add link and fix the other one. Otherwise lgtm.

File: aio/content/guide/browser-support.md (hunk 1 above)
Comment (brandonroberts) [reply aio/content/guide/browser-support.md]: The links are updated now.

**Pull request closed without merging**